### PR TITLE
Scoreboard feature added

### PR DESCRIPTION
**Pull request by dmitry-vereykin**
**Description:**

Added a list that represents the scoreboard to "pug" template file. Visual representation only, functionality can be easily added by linking to app.js

**Testing:**

No requirements - simply run app.js with "node" or "nodemon."

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+var express = require('express');
+var app = express();
+
```
Comment (AaronPNg) [inline app.js:3]: no need for white space here

Comment (dmitry-vereykin) [reply app.js:3]: I disagree with you good Sir. Extra white space provides better readability.

**Comment by AaronPNg** [comment]
good use of white space but some can be omitted

File: app.js (hunk 1 above)
Comment (dmitry-vereykin) [reply app.js:3]: 79a

**Pull request closed without merging**